pyaedt/modules/PostProcessor.py

```diff
@@ -1642,7 +1644,7 @@ def create_report(
             The default is ``None``. It can be `None`, `"Differential Pairs"`,`"RL"`,
             `"Sources"`, `"Vias"`,`"Bondwires"`, `"Probes"` for Hfss3dLayout or
             Reduce Matrix Name for Q2d/Q3d solution or Infinite Sphere name for Far Fields Plot.
```

Comment:
```suggestion
            The default is ``None``.
            
            - For HFSS 3D Layout, options are ``"Bondwires"``, ``"Differential Pairs"``,
              ``None``, ``"Probes"``, ``"RL"``, ``"Sources"``, and ``"Vias"``.
            - For Q2D or Q3D, specify the name of a reduced matrix.
            - For a far fields plot, specify the name of an infinite sphere.
```
There is no brief description for this parameter. I rewrote the options for clarity. However, this implies that "None" is the default for any of the bulleted list entries. Is this true. This may need to be edited further.